po/zh_CN.po

```diff
@@ -0,0 +1,1948 @@
+# Simplified Chinese translations for com.github.johnfactotum.Foliate package.
+# Copyright (C) 2019 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the com.github.johnfactotum.Foliate package.
+# Blanc Chan cdh4227@qq.com, 2020.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: com.github.johnfactotum.Foliate\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-04-21 15:05+0800\n"
+"PO-Revision-Date: 2020-05-12 13:55+0800\n"
+"Last-Translator: Blan Chan cdh4227@qq.com\n"
+"Language-Team: none\n"
+"Language: zh_CN\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:3 src/main.js:105
+#: data/com.github.johnfactotum.Foliate.appdata.xml.in:6
+msgid "Foliate"
+msgstr ""
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:4
+msgid "eBook Viewer"
+msgstr "电子书查看器"
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:5
+msgid "View eBooks"
+msgstr "阅读电子书"
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:9
+msgid "com.github.johnfactotum.Foliate"
+msgstr ""
+
+#: data/com.github.johnfactotum.Foliate.desktop.in:12
+msgid "Ebook;Book;EPUB;Viewer;Reader;"
+msgstr "电子书;图书;EPUB;查看器;阅读器"
+
+#: src/main.js:80
+msgid "All Files"
+msgstr "所有文件"
+
+#: src/main.js:84
+msgid "E-book Files"
+msgstr "电子书文件"
+
+#: src/main.js:90
+msgid "Open File"
+msgstr "打开文件"
+
+#: src/main.js:104#, fuzzy
+msgid "translator-credits"
+msgstr "翻译者致谢"
```

Comment:
The `translator-credits` string is used to credit translators in the "About" dialog. Should contain a list of translators' names separated by `\n` instead of the literal translation of "translator-credits".